src/typescript/azure-openapi-validator/rules/DescriptionTooShort.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+import { trimDescription } from './utilities';
+
+export const DescriptionTooShort: string = "DescriptionTooShort";
+const minimumDescriptionLength = 20;
+
+rules.push({
+  id: "D4002",
+  name: DescriptionTooShort,
+  severity: "error",
```

Comment:
Do we really want this to be an error?

Reply:
See above regarding my comment about short descriptions often lacking contextual information. I'd be willing to introduce a new `variable` severity type (where the exact severity is returned in the yielded response.) Then abnormally short descriptions could be errors, and descriptions under a "reasonable" length would be warnings. Future proposed rules will also require variable severity.

Reply:
Please confirm with @kirthik